src/explore-education-statistics-frontend/src/modules/table-tool/components/TableTool.tsx

```diff
@@ -40,6 +44,18 @@ const TableToolFinalStep = ({
     setPermalinkId('');
   }, [tableHeaders, table]);
 
+  useEffect(() => {
+    const setGoToMethodologyLinkData = async () => {
+      if (publication) {
+        const response = await publicationService.getLatestPublicationRelease(
+          publication.slug,
+        );
+        setMethodologyLink(response.publication);
```

Comment:
This is actually setting the release rather than the methodology link and should be named as such as we are adding in potential confusion here.

Reply:
Done